starknet_py/net/full_node_client.py

```diff
@@ -65,10 +66,14 @@ def __init__(
         """
         self.url = node_url
         self._client = RpcHttpClient(url=node_url, session=session)
+
+        if net is not None:
+            warnings.warn("Parameter net is deprecated.", category=DeprecationWarning)
         self._net = net
 
     @property
-    def net(self) -> Network:
+    def net(self) -> Optional[Network]:
+        warnings.warn("Parameter net is deprecated.", category=DeprecationWarning)
```

Comment:
This doesn't tell which class doesn't accept the parameter anymore. Did you check manually how does the trace look when using the library with `net`, and if user can tell where the `warn` has its' origin?